shared/src/main/java/gov/hhs/cdc/trustedintermediary/utils/RetryFailedException.java

```diff
@@ -0,0 +1,11 @@
+package gov.hhs.cdc.trustedintermediary.utils;
+
+public class RetryFailedException extends Exception {
```

Comment:
We should write a test for this.  See something like `HttpClientExceptionTest` for an example.